EU4toV2/Source/V2World/Decisions/Decisions.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Decisions.h"
+#include "ParserHelpers.h"
+#include "OSCompatibilityLayer.h"
+#include "Log.h"
+
+V2::Decisions::Decisions(const std::string& filename)
+{
+	registerKeyword("political_decisions", [this](const std::string& unused, std::istream& theStream) {
+		Decisions newDecisions(theStream);
+		decisions = newDecisions.getDecisions();
+	});
```

Comment:
add a catchall beneath here.

Reply:
you also want to fix travis. Add the new files to CMakeLists.

Reply:
> you also want to fix travis. Add the new files to CMakeLists.

I thought I did here [https://github.com/ParadoxGameConverters/EU4toVic2/pull/599/commits/d98a52092c265d41e4d59d3c883b7a94e1e4b0b6](url)